src/repository/DeviceKeyFileRepository.js

```diff
@@ -13,31 +13,20 @@ class DeviceKeyFileRepository {
     this._fileManager = new FileManager(path);
```

Comment:
Can you add `implements IRepository` to this?

Also, do we not need the `create` method anymore? I really think we should keep it even though it does the same thing as `update`. There are some cases where there will be different behaviors between the two functions when someone overrivdes this for APIs.